src/deadline/nuke_submitter/default_nuke_job_template.yaml

```diff
@@ -25,6 +25,11 @@ parameterDefinitions:
     label: Gizmo Directory
   description: The directory containing Nuke Gizmo files used by this job.
   default: './gizmos'
+- name: OCIOConfigPath
+  type: PATH
+  objectType: FILE
+  dataFlow: IN
+  description: The OCIO config file used by this job.
```

Comment:
Is there a default that can make sense here?

Reply:
Someone else can correct me if I'm wrong. 🙏 
I don't think we have a perfect default path T-T because it's really up to customers. Customers can bring their configuration by themselves or use Nuke's default configurations where Nuke's default has more than 3 configuration that customers can choose so it is hard to choose one.

Reply:
Makes sense 👍